webapp/src/ts/modules/contacts/contacts-edit.component.ts

```diff
@@ -29,6 +30,7 @@ export class ContactsEditComponent implements OnInit, OnDestroy, AfterViewInit {
     private contactTypesService:ContactTypesService,
     private dbService:DbService,
     private contactSaveService:ContactSaveService,
+    private telemetryService: TelemetryService,
```

Comment:
Nit: whitespace consistency.

```suggestion
    private telemetryService:TelemetryService,
```